docker/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM archlinux/base:latest
+LABEL Alad Wenter <alad@archlinux.org>
+
+RUN pacman -Syu --noconfirm base base-devel
+RUN pacman -Syu --noconfirm jq pacutils git wget shellcheck vim vifm devtools bash-completion parallel
+RUN ln -fs /usr/share/zoneinfo/Europe/Berlin /etc/localtime
+
+# unprivileged user
+ARG USERNAME=archie
+RUN useradd -m -s /bin/bash $USERNAME
+RUN install -o $USERNAME -dm755 /home/$USERNAME/.local
+RUN install -o $USERNAME -dm755 /home/$USERNAME/.local/bin
```

Comment:
Every `RUN` creates a new image layer, I would suggest optimizing stuff a little bit, e.g. merging these two `RUN` into one, and above two `pacman` installations - don't go crazy trying to minimize the number of layers, but in this case I just think there is no value to create two layers, one without `bin` folder yet created and another one with.